vendor/src/github.com/docker/engine-api/types/volume/volume.go

```diff
@@ -0,0 +1,27 @@
+package volume
+
+// Volume represents the configuration of a volume for the remote API
+type Volume struct {
+	Name       string                 // Name is the name of the volume
+	Driver     string                 // Driver is the Driver name used to create the volume
+	Mountpoint string                 // Mountpoint is the location on disk of the volume
+	Status     map[string]interface{} `json:",omitempty"` // Status provides low-level status information about the volume
+	Labels     map[string]string      // Labels is metadata specific to the volume
+	Scope      string                 // Scope describes the level at which the volume exists (e.g. `global` for cluster-wide or `local` for machine level)
```

Comment:
Scopes need to be able to represent different kinds of network horizons. The examples imply that global and local are the only options.